runtime-decompiler/src/main/java/org/jrd/frontend/MainFrame/MainFrameView.java

```diff
@@ -93,8 +93,10 @@ protected void processMouseMotionEvent(MouseEvent e) {
         localVmList.addMouseListener(new MouseAdapter() {
             @Override
             public void mouseClicked(MouseEvent mouseEvent) {
-                ActionEvent event = new ActionEvent(localVmList, 0, null);
-                vmChangingListener.actionPerformed(event);
+                if(mouseEvent.getButton() == MouseEvent.BUTTON1){
+                    ActionEvent event = new ActionEvent(localVmList, 0, null);
+                    vmChangingListener.actionPerformed(event);
+                }
```

Comment:
Is this applicable to remotes too?

Maybe this can be even better, and relod the list only if the proces really changes? (I believe it have selection change listener)

Reply:
Good point, tho I haven't experimented with remote processes yet. Will have to see.